content/blog/eng/v-protetto8-3-2024.md

```diff
@@ -0,0 +1,126 @@
+- We asked the challengers to form *4-persons teams*
+- The challenge is: *how many point you can get in 2.5 hours*?
+- and that's it!
+
+Leaderboard manage how the points are computed and all other tedious stuff.
+
+All teams are free to solve the puzzles in the order they want (and this was crucial for the winning team!), keeping in mind that the difficulty is ramping up from day 1 and soon you can get stuck in such a strict time-box.
+
+![Wheel of Years](/images/code-challenge-march-2024/wheel-of-years.png "If you have to extract something randomly in public, why don't you do in a fancy way, like the good ol' Wheel of Fortune?")
+*If you have to extract something randomly, why don't you do in a fancy way, like the good ol' [Wheel of Fortune?](https://wheelofnames.com/)*
+
+# Suit up!
+
+6 teams were formed.
+They were `stderr`, `Tori Automatici` ("automated bulls"), `Ittasinanta`, `Loans&Lions`, `Daje` ("go!" in roman dialect), `Alco1isti Anon1mi`.
+
+ The teams _in some quite eterogeneous way_, in fact they were composed in the same team by backend developers, frontend ones, data engineers, qa analysts and so on. 
+
+Most of the participants were _newcomers_, they never made an Advent of Code before, and this was the main difficulty of some teams, one specifically. Other ones are made of veterans of this challenge, and they proceed like a wind.
+
+## On you mark, ready, steady, go!
+
+The challenge was very intense, and until the last 10 minutes of the challenge, it was an head to head to the first two teams.
+
+Every single team used a [mob programming technique](https://en.wikipedia.org/wiki/Team_programming#Mob_programming) way of solve problems: _the entire team_ focus itself _on a single problem_. One person actively writes the code, others suggesting solutions. This is a very interesting way to share knowledge, expecially in a never-tried setting. This is also very useful to make team building.
+
+All teams... but one. _The winning one_.
+The winning team (mainly composed of colleague from the same area) used a different approach: everyone takes a problem and try to solve it. They switch on mob programming only to solve difficult issues. This approach let they get the latest winning point solving a problem a litle further from the one everyone were stuck.
+
+## Wrapping up
+
+Here it is, the final score of the challenge.
+
+![Leaderboard Advent Of Code March 2024](/images/code-challenge-march-2024/leaderboard-code-challenge.png "Congrats to Daje Team!")
+*All glory to Daje team!*
+
+Besides of the mere numbers, everyone enjoys the day, considering it though but fun. _Also, it helps people from different teams_ - and with different expertise - _wrapping up together_, knowing each others.
+
+The challenge sharpen up collaboration and the strict time-box helps the team to focus on what matters. The side effect is, when the time almost expires, someone gave up because it was unable to focus anymore.
```

Comment:
- The challenge ~sharpen up~ collaboration and the strict ~time-box~ ~helps~ the team to focus on what ~matters. -> The challenge **sharpened** the collaboration and the strict **time box** **helped** the team to focus on what matters. 
- The side effect ~is~, when the time almost ~expires~, someone gave up because ~it~ was unable to focus anymore. ->  The side effect **was** **that** when the time almost **expired**, someone gave up because **they** **were** unable to focus anymore.